test/distributed/org/apache/cassandra/distributed/fuzz/HarryHelper.java

```diff
@@ -26,19 +26,26 @@
 import harry.model.clock.OffsetClock;
 import harry.model.sut.PrintlnSut;
 
+import static org.apache.cassandra.config.CassandraRelevantProperties.CASSANDRA_ALLOW_SIMPLESTRATEGY;
+import static org.apache.cassandra.config.CassandraRelevantProperties.CASSANDRA_MINIMUM_REPLICATION_FACTOR;
+import static org.apache.cassandra.config.CassandraRelevantProperties.DISABLE_TCACTIVE_OPENSSL;
+import static org.apache.cassandra.config.CassandraRelevantProperties.LOG4J2_DISABLE_JMX;
+import static org.apache.cassandra.config.CassandraRelevantProperties.LOG4J2_SHUTDOWNHOOKENABLED;
+import static org.apache.cassandra.config.CassandraRelevantProperties.ORG_APACHE_CASSANDRA_DISABLE_MBEAN_REGISTRATION;
+
 public class HarryHelper
 {
     public static void init()
     {
-        System.setProperty("log4j2.disableJmx", "true"); // setting both ways as changes between versions
-        System.setProperty("log4j2.disable.jmx", "true");
-        System.setProperty("log4j.shutdownHookEnabled", "false");
-        System.setProperty("cassandra.allow_simplestrategy", "true"); // makes easier to share OSS tests without RF limits
-        System.setProperty("cassandra.minimum_replication_factor", "0"); // makes easier to share OSS tests without RF limits
-
-        System.setProperty("cassandra.disable_tcactive_openssl", "true");
-        System.setProperty("relocated.shaded.io.netty.transport.noNative", "true");
-        System.setProperty("org.apache.cassandra.disable_mbean_registration", "true");
+        // setting both ways as changes between versions
+        System.setProperty("log4j2.disableJmx", "true"); // checkstyle: suppress nearby 'blockSystemPropertyUsage'
+        LOG4J2_DISABLE_JMX.setBoolean(true);
+        LOG4J2_SHUTDOWNHOOKENABLED.setBoolean(false);
```

Comment:
I've updated this part of the code, but here are my thoughts:

The documentation pages say:
https://logging.apache.org/log4j/2.x/manual/configuration.html

The properties have changed in Log4j 2.10, and all system property names have been normalized to follow a consistent naming scheme. While the old property names are still supported for backwards compatibility:

- the `log4j2.disableJmx` and the legacy property name`log4j2.disable.jmx`;
- the `log4j2.shutdownHookEnabled` and the legacy property name `log4j.shutdownHookEnabled`;

Since a well-known 0-day vulnerability in the Log4j, I think most of the users are using the latest library version, thus we don't need to add legacy property names to the `CassandraRelevantProperties`. Actually, I think they can be removed, but  I'm afraid of doing such changes in this PR.

Reply:
you are all right here but I would leave this to another ticket, this work is just about making the heavy lifting in the first place so lets just do that and then we can contemplate about changing this. It may seem like unnecessary exercise but I would really dedicate this to separate ticket for better visibility otherwise this will be burried under all other changes ....

Reply:
Ok, it makes sense to me.
I will file an issue right after we'll finish with the current changes.